pyrdp/mitm/MCSMITM.py

```diff
@@ -81,7 +81,8 @@ def onConnectInitial(self, pdu: MCSConnectInitialPDU):
 
         rdpClientDataPDU = rdpClientConnectionParser.parse(gccConferenceCreateRequestPDU.payload)
 
-        if "MS_T120" in map(lambda channelDef: channelDef.name, rdpClientDataPDU.networkData.channelDefinitions):
+        if rdpClientDataPDU.networkData is not None and\
```

Comment:
This attribute is already validated at line 112. You should move your code there.

Reply:
Nice catch, will do.

Reply:
done